src/tox_uv/_venv.py

```
@@ -83,6 +86,18 @@ def _get_python(self, base_python: list[str]) -> PythonInfo | None:  # noqa: PLR
                     platform=sys.platform,
                     extra={},
                 )
+            if os.path.isabs(base):  # noqa: PTH117
```

Why not use pathlib?

---

Not constructing or modifying the path at all, just checking its absolute-ness. No value add in constructing `Path` objects.

---

Well, there is that value that you don't have to silence the error and Lower the cognitive overload of what this error means when someone reads it. I personally would just use it and not override the error.

---

Sure. Amended.